scripts/dts/edtlib.py

```diff
@@ -1297,6 +1314,9 @@ def _set_instance_no(self):
 
         self.instance_no = {}
 
+        if not self.enabled:
+            return
+
```

Comment:
Does this make a difference for the output? `gen_defines.py` only seems to look at `node.instance_no` for enabled nodes.

Reply:
Short history...the check above is necessary to keep `dt_compat_get_str` given same result of  `gen_defines.py`.